<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpticalMarginEditor</class>
 <widget class="QWidget" name="OpticalMarginEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Set:</string>
       </property>
       <property name="buddy">
        <cstring>comboSets</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboSets">
       <property name="toolTip">
        <string>Current selected rule set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonResetSet">
       <property name="toolTip">
        <string>Reset current set to default values.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonAddNewSet">
       <property name="toolTip">
        <string>Add new rule set</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonRemoveSet">
       <property name="toolTip">
        <string>Remove current selected rule set</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sets:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonResetAllSets">
       <property name="toolTip">
        <string>Load default rule sets. Replaces current rule sets</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonLoadMissingDefaults">
       <property name="toolTip">
        <string>Add missing default sets to current sets.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="buttonAddNewRow">
       <property name="toolTip">
        <string>Add a new rule</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="labelUnicode">
       <property name="text">
        <string>Unicode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUnicodePreview">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboSets</tabstop>
  <tabstop>buttonAddNewSet</tabstop>
  <tabstop>buttonRemoveSet</tabstop>
  <tabstop>buttonResetAllSets</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>buttonAddNewRow</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
